### PR TITLE
COVID-19 Table and Page patch

### PR DESCRIPTION
**Pull request by jcheadle-rti**
Addressing comments from Becky Boyles, Steph Suber, and Lauren Hochman regarding edits to the content of the COVID-19 table and page.

https://docs.google.com/document/d/1sy8FUc2cXZLd-4n7WRdvYeEx4R-qQBvKxEWnX6Dj6Cg/edit?usp=sharing

### CODE REVIEW

File: src/data/studies/covid-studies-columns.json [hunk 1]
```diff
@@ -49,5 +56,12 @@
...
-]
\ No newline at end of file
+  },
+  {
+    "name": "Other Information",
+    "selector": "Other_Information",
+    "type": "string",
+    "omit": true,
+    "groupable": false
+  },
```
Comment (mbwatson) [inline src/data/studies/covid-studies-columns.json:66]: can you remove this trailing comma?

File: src/data/studies/covid-studies.json [hunk 2]
```diff
@@ -3,10 +3,12 @@
     "Name": "Outcomes Related to COVID-19 Treated With Hydroxychloroquine Among In-patients With Symptomatic Disease (ORCHID)",
     "Short Name": "ORCHID",
     "Description": "ORCHID is a multicenter, blinded, placebo-controlled, randomized clinical trial evaluating hydroxychloroquine for the treatment of adults hospitalized with COVID-19. Patients, treating clinicians, and study personnel will all be blinded to study group assignment.",
+    "dbGaP Accession ID": "phs002299.v1.p1",
     "Type": "Interventional (Clinical Trial)",
     "Link": "https://clinicaltrials.gov/ct2/show/NCT04332991",
     "Network": "PETAL Network",
     "Responsible Party": "Boyd Taylor Thompson, Massachusetts General Hospital",
-    "Status": "Completed"
+    "Status": "Completed",
```
Comment (mbwatson) [inline src/data/studies/covid-studies.json:11]: can you remove this trailing comma?

**Review by jcheadle-rti — COMMENTED** [review]
Removed trailing commas per suggestion

File: src/data/studies/covid-studies.json [hunk 3]
```diff
@@ -3,10 +3,12 @@
...
+    "Other Information": "Access to ORCHID data requires data access approvals from the NLM Database of Genotypes and Phenotypes (dbGaP) system. ORCHID data is also available to those with appropriate access permission on BioLINCC (https://biodatacatalyst.nhlbi.nih.gov/data-protection), where they may download the data.",
```
Comment (mbwatson) [inline src/data/studies/covid-studies.json]: another trailing comma. please remove :)

Comment (jcheadle-rti) [reply src/data/studies/covid-studies.json]: I edited directly and it says 'outdated' right now - the trailing comma appears to be removed from the 'Other Information' line
image

**Pull request merged**